module/documents/actor/actor.mjs

```diff
@@ -1709,6 +1709,64 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
 
   /* -------------------------------------------- */
 
+  /**
+   * Perform a saving throw to maintain concentration.
+   * @param {object} [options]          Options to configure how the saving throw is rolled
+   * @returns {Promise<D20Roll|null>}   A Promise which resolves to the created Roll instance
+   */
+  async rollConcentration(options={}) {
+    if ( !this.isOwner ) return null;
+    const conc = this.system.attributes?.concentration;
+    if ( !conc ) throw new Error("You may not make a Concentration Saving Throw with this Actor.");
+
+    const config = CONFIG.DND5E;
+    const modes = CONFIG.Dice.D20Roll.ADV_MODE;
+    const parts = [];
+
+    // Concentration bonus
+    if ( conc.bonus ) {
+      parts.push(conc.bonus);
+    }
+
+    const ability = (conc.ability in config.abilities) ? conc.ability : config.defaultAbilities.concentration;
+
+    options = foundry.utils.mergeObject({
+      isConcentration: true,
+      targetValue: 10,
+      advantage: options.advantage || (conc.roll.mode === modes.ADVANTAGE),
+      disadvantage: options.disadvantage || (conc.roll.mode === modes.DISADVANTAGE),
```

Comment:
I might have missed something but I think `mode` is a top level property of `RollConfigField`:

```suggestion
      advantage: options.advantage || (conc.mode === modes.ADVANTAGE),
      disadvantage: options.disadvantage || (conc.mode === modes.DISADVANTAGE),
```

Maybe it should be under `roll`?

Reply:
We could maybe discuss that on my PR to add the `RollConfig` field to other areas of the actor model, but I'm not opposed. The original idea was `roll` being *modifiers* for the roll, but I suppose that advantage mode very much is that as well.